package com.cetpainfotech.day9;

public class ExceptionDemo4 {
	public static void main(String[] args) {
		
		try
		{
			int a = 6;
			int b =2;
			int c = a/b;
			
			int arr[] = {1,2,4};
			System.out.println(arr[3]);
			
			String str = null;
			System.out.println(str.toCharArray());
			
		 }catch (ArithmeticException e) {
			
			 System.out.println("Devide by zero");
		}catch (ArrayIndexOutOfBoundsException e) {
			
			System.out.println("Array Index not found");
		}
		
		System.out.println("End");
		
	}

}
